mail/astroid/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        astroidmail astroid 0.15 v
+
+description         Astroid Mail
+long_description    A graphical threads-with-tags style, lightweight and fast, e-mail client for Notmuch
+maintainers         @arietis
```

Comment:
Normally the maintainer block would be set to your name + `openmaintainer`, indicating that others can contribute to this port as well.

```
maintainers @arietis \
            openmaintainer
```

See: https://guide.macports.org/chunked/reference.html#reference.keywords.maintainers